src/imitation_loss/imitation_loss.jl

```diff
@@ -0,0 +1,76 @@
+"""
+    ImitationLoss{L,R,P}
+
+Generic imitation loss: max_y ℓ(y, y̅) + α θᵀ(y - y̅) - (Ω(y) - Ω(y̅)).
```

Comment:
Fix docstrings to reflect that `t_true` is now accepted instead of `y_true`

Reply:
Notations should be consistent with argument names: either a bar or a `_true` suffix